lockup/program/Cargo.lock

```diff
@@ -1,2469 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
Removing these lock files instead of updating them, since these crates will be removed in https://github.com/project-serum/serum-dex/pull/95.